### DISCUSSION TITLE
Developer experience for the Mesh for Data

### DISCUSSION THREAD

**Discussion by ronenkat** [fybrik/fybrik#245] [Developers]
@hunchback @roee88 Following #233 and my own attempts to build a dev environment, I want to discuss how a developer engages with the mesh for data.
Current the website have for code contribution:
* Contribution Flow
** explain what build and test does....
* Development Environment

I would like to suggest the following structure to define how to engage, which will also reflect on content and scripts.
* Development Environment requirements
* Contribution Flow (subset of the current one)
* Running and testing on a cluster
** setup your own cluster using Kind
* Running in a multi cluster env

I would like to adapt our scripts and make to reflect these options.
Please let me know your thoughts.

**Comment by roee88** [comment]
sounds good to me.

> I would like to adapt our scripts and make to reflect these options.

I assume this is only relevant to the last two items in your list. What are you proposing here exactly compared to what we already have? That is, what should change?

**Reply by ronenkat** [reply]
Basically yes, it should impact the way we run tests, and it should be useful for developing the data mesh.
It didn't make sense to me reading it --- and I am aiming to offer suggestion after more experimentation.
